portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Billy Porter</h1>
```

Comment:
Is this file still in use?

Reply:
This was part of the tutorial for Week 2. It is in use for that starter website but not for my Angular version.